extensions/resteasy-reactive/rest-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/SimpleJsonWithReflectionFreeSerializersTest.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.resteasy.reactive.jackson.deployment.test;
+
+import java.util.function.Supplier;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.security.test.utils.TestIdentityController;
+import io.quarkus.security.test.utils.TestIdentityProvider;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class SimpleJsonWithReflectionFreeSerializersTest extends SimpleJsonTest {
+
+    @RegisterExtension
+    static QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(new Supplier<>() {
+                @Override
+                public JavaArchive get() {
+                    return ShrinkWrap.create(JavaArchive.class)
+                            .addClasses(Person.class, SimpleJsonResource.class, User.class, Views.class, SuperClass.class,
+                                    OtherPersonResource.class, AbstractPersonResource.class, DataItem.class, Item.class,
+                                    NoopReaderInterceptor.class, TestIdentityProvider.class, TestIdentityController.class,
+                                    AbstractPet.class, Dog.class, Cat.class, Veterinarian.class, AbstractNamedPet.class,
+                                    AbstractUnsecuredPet.class, UnsecuredPet.class, SecuredPersonInterface.class, Frog.class,
+                                    Pond.class, FrogBodyParts.class, FrogBodyParts.BodyPart.class)
+                            .addAsResource(new StringAsset("admin-expression=admin\n" +
+                                    "user-expression=user\n" +
+                                    "birth-date-roles=alice,bob\n" +
+                                    "quarkus.jackson.optimization.enable-reflection-free-serializers=true\n"),
```

Comment:
I made the generation of reflection-free serializers optional and disabled by default. Then I wanted all the tests in `SimpleJsonTest` to be executed both with and without those serializers. The solution that I found is creating another test class that extends `SimpleJsonTest` and rewriting the entire `QuarkusUnitTest` configuration adding the properties to enable the serializers generation. This works as expected, but I wonder if there is any more elegant/concise way to achieve the same without duplicating the entire `QuarkusUnitTest` definition. /cc @geoand

Reply:
I don't think there is much else you can do